generate.2.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-BASE=$(pwd)
-
-while getopts g:a:t:p:o:v:m:u: option
+while getopts g:a:t:p:o:v:m:u:w: option
```

Comment:
anyways, this is still a work in progress. much more changes needed. any other suggestions for the changes made till now?